### PR TITLE
Add usage and CI

### PR DESCRIPTION
**Pull request by kenji-miyake**

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
--   id: prettier-xml
-    name: Prettier XML
-    description: Apply prettier to package.xml
-    entry: prettier --write --list-different --ignore-unknown --print-width 1000 --xml-self-closing-space false
-    language: node
-    files: package.xml$
-    # Use plugin-xml@0.13.1 for xml-self-closing-space option
-    additional_dependencies: ["prettier@2.3.2", "@prettier/plugin-xml@0.13.1"]
+- id: prettier-xml
```
Comment (KeisukeShima) [inline .pre-commit-hooks.yaml:1]: pre-commitドキュメントの例ではスペース4個となっているのですが、スペースを2個にした理由はありますか？

https://pre-commit.com/#creating-new-hooks

Comment (kenji-miyake) [reply .pre-commit-hooks.yaml:1]: I saw these files and thought 2-spaces is okay.
https://github.com/gruntwork-io/pre-commit/blob/master/.pre-commit-hooks.yaml
https://github.com/hadolint/hadolint/blob/master/.pre-commit-hooks.yaml

If you use `prettier` and `markdownlint` with 4-spaces, it causes some problems and we must set some options.
https://github.com/DavidAnson/markdownlint/commit/580e57ddeec0d59e65b2872a7e70622fb03858df

To keep it simple, I think 2-spaces is better.

Comment (KeisukeShima) [reply .pre-commit-hooks.yaml:1]: I understand. That makes sense.

**Review by KeisukeShima — APPROVED** [review]
LGTM :+1:

**Pull request merged**